### DISCUSSION TITLE
add 3 axis Translate gizmo to Path points

### DISCUSSION THREAD

**Discussion by Miziziziz** [godotengine/godot-proposals#4613] [Editor]
Currently it is very awkward to move path points to the positions you want them to be as they only move relative to camera angle. 
![image](https://user-images.githubusercontent.com/7292421/171487177-061b5a40-f64d-45e2-aef2-b1954f56c68d.png)

It would be nice if you could move them with the standard translate gizmo:
![image](https://user-images.githubusercontent.com/7292421/171487338-6e04da85-9d3b-46d9-9430-21ba018b13e2.png)

maybe you click on a point and it turns into the gizmo and then turns back into a point if you click on something else.

**Comment by aXu-AP** [comment]
I've recently worked on a few addons which use Path, and I have to say, this would be a great addition.
Other things which would help:
- Snap to colliders. Currently when adding points to path they get created on fixed distance from camera and it's hard to gauge where. Having them appear on floor or whatever is under the mouse would be great.
- Make curves by dragging. I don't know why you need to put points first and add curve handles afterwards.
- Make curve in/out handles less intruding. They are quite overpowering as they consist 2/3 of the handles while being less important. Maybe smaller handles or make them disappear until associated path point is clicked.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/527.
